README.md

```diff
@@ -2,63 +2,40 @@
 
 Django-Todolist is a todolist web application with the most basic features of most web apps, i.e. accounts/login, API and (somewhat) interactive UI.
 
----
-CSS | [Skeleton](http://getskeleton.com/)
-JS  | [jQuery](https://jquery.com/)
+## Instructions for running MySQL container with a volume attached
 
-## Explore
-Try it out by installing the requirements. (Works only with python >= 3.8, due to Django 4)
-
-    pip install -r requirements.txt
-
-Create a database schema:
-
-    python manage.py migrate
-
-And then start the server (default: http://localhost:8000)
+1. Pull the MySQL image and run a container with a volume attached using the following command:
+    ```bash
+    docker run -d -v mysql_data:/var/lib/mysql --name mysql-container -e MYSQL_ROOT_PASSWORD=1234 -e MYSQL_DATABASE=app_db -e MYSQL_USER=app_user -e MYSQL_PASSWORD=1234 mysql-local:1.0.0
```

Comment:
```suggestion
    docker run -d -v mysql_data:/var/lib/mysql --name mysql-container
```
No need to explicitly define env variables as flags for run command assuming that you defined all required envs inside of Dockerfile.mysql. This is not a good practice.